plugins/eslint-config/index.js

```diff
@@ -43,6 +43,7 @@ module.exports = {
       tagNamePreference: {
         'returns': 'return',
       },
+      mode: 'closure',
```

Comment:
Double-checking: is this supposed to be in this change?

Reply:
Yes, because somewhere in here I have a method where I need to set `this` in the jsdoc, and the format of setting that is different depending on whether or not you are in closure mode.